firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -77,6 +77,15 @@
           .setDownloadUrl("https://test-url")
           .build();
 
+  private static final AppDistributionReleaseInternal TEST_RELEASE_NEWER_AAB_INTERNAL_2 =
```

Comment:
I don't think we need to create a second object for testing. If we turn these into builder objects (AppDistributionReleaseInternal.Builder), then the tests can write:

testReleaseNewerAabInternal.build() // no release notes
testReleaseNewerAaabInternal.setReleaseNotes(....).build() // with release notes

This blog has a good explanation of the design pattern: https://testing.googleblog.com/2018/02/testing-on-toilet-cleanly-create-test.html 

(If you adopt this pattern, we may want to change these private fields into private methods that return a builder when called. )